native/cocos/platform/interfaces/modules/ISystemWindow.h

```diff
@@ -89,6 +89,7 @@ class CC_DLL ISystemWindow : public OSInterface {
     }
     virtual uintptr_t getWindowHandler() const = 0;
     virtual Size      getViewSize() const      = 0;
+    virtual void      setViewSize(uint32_t width, uint32_t height) = 0;
```

Comment:
Is it better to write a non-pure virtual function here?

Reply:
Possibly, I can change it if you want